lib/pengine/bundle.c

```diff
@@ -1509,7 +1509,7 @@ pe__bundle_xml(pcmk__output_t *out, va_list args)
 
     print_everything = pcmk__str_in_list(only_rsc, rsc->id);
 
-    for (GList *gIter = bundle_data->replicas; gIter != NULL;
+    for (GListPtr gIter = bundle_data->replicas; gIter != NULL;
```

Comment:
I don't mind using xmlNodePtr since it's provided by libxml2, but I'm not a fan of GListPtr because we define that ourselves, and it's kind of stepping on glib's namespace (not to mention inconsistent since we don't define Ptrs for any other types). I'd rather go in the other direction with that one.

Reply:
Fixed.